modules/metering-store-data-in-shared-volumes.adoc

```diff
@@ -36,6 +36,6 @@ spec:
 
 There are two options to modify your configuration file:
 
-<1> Set `storage.hive.sharedPVC.claimName` to the name of an existing `ReadWriteMany` persistent volume claim (PVC). This is necessary if you do not have dynamic volume provisioning, or want to have more control over how the `PersistentVolume` is created.
+<1> Set `storage.hive.sharedPVC.claimName` to the name of an existing ReadWriteMany PersistentVolumeClaim (PVC). This is necessary if you do not have dynamic volume provisioning, or want to have more control over how the PersistentVolume is created.
```

Comment:
I'd make it clear in these annotations that you need to pick one one.
(no comma)